caddytls/setup.go

```diff
@@ -66,6 +66,12 @@ func setupTLS(c *caddy.Controller) error {
 		for c.NextBlock() {
 			hadBlock = true
 			switch c.Val() {
+			case "ca":
+				arg := c.RemainingArgs()
```

Comment:
We should use only the first argument; if there are more or less than 1 argument, it should be an error.